.travis.yml

```diff
@@ -112,18 +112,22 @@ matrix:
       script:
         - "./build/travis/job_macos/script.sh"
 
-
 notifications:
-  recipients:
-    - $NOTIFICATION_EMAIL
   email:
+    recipients:
+      - "build@musescore.org"
     on_success: change
     on_failure: always
   irc:
     channels:
-      - $NOTIFICATION_IRC
+      - "chat.freenode.net#musescore"
     on_success: change
     on_failure: change
+    template: # https://docs.travis-ci.com/user/notifications/#IRC-notification
```

Comment:
Too many lines, or the line itself is too long? If it's the latter then it's not actually that long once the variables get substituted in. E.g.:

<pre>
Build #174 for musescore/MuseScore failed after 2 min 46 sec. Author: shoogle
Branch: fix-travis-notifications, Commit: fb007e0 "New syntax for travis notifications"
Changes: https://github.com/shoogle/MuseScore/compare/09631ec750f1...fb007e0ac57d
Build log: https://travis-ci.org/shoogle/MuseScore/builds/142512037
</pre>

Travis' default notification template is:
<pre>
musescore/MuseScore#174 (fix-travis-notifications - fb007e0 - shoogle): The build failed.
Change view: https://github.com/shoogle/MuseScore/compare/09631ec750f1...fb007e0ac57d
Build details: https://travis-ci.org/shoogle/MuseScore/builds/142512037
</pre>